package.json

```diff
@@ -8,9 +8,12 @@
   "repository": "https://github.com/thysultan/stylis.js",
   "bugs": "https://github.com/thysultan/stylis.js/issues",
   "type": "module",
-  "main": "dist/stylis.umd.js",
+  "main": "dist/umd/stylis.umd.js",
```

Comment:
Shouldn't commonjs modules have .cjs extension? I think this is a requirement when "type" is "module". The way around is package.json with "{ type: "commonjs" }" in subfolder.

Reply:
@TrySound - see a comment [here](https://github.com/thysultan/stylis.js/pull/207#discussion_r407931392) 😉

Reply:
Nice!

Reply:
Why the new directory?

Reply:
To add another package.json with { type: commonjs }